### DISCUSSION TITLE
Roblox player instantly terminates while sandboxed

### DISCUSSION THREAD

**Discussion by qwoahtrevor** [sandboxie-plus/Sandboxie#4069] [General]
I've messed around with it for a while and gotten Roblox to properly install (as far as I am aware, nothing wrong went wrong with the install) within a sandbox. The problem is, whenever it is launched it instantly terminates. I don't have this problem for any other programs, just Roblox. I tried Roblox studio too and it doesn't work. Maybe it's a problem with Roblox, or maybe it's intentional? I don't really know. I can provide more details if needed and I've experimented with a few things. An alternative idea I had was to try and get the Microsoft Store version of Roblox installed on the sandbox but have so far found it more challenging than I had hoped. If anyone has any ideas on how to fix this behavior or get the Microsoft Store version of Roblox installed onto a sandbox please let me know.

**Comment by bastik-1001** [comment]
Until #19 gets resolved, UWP/Store apps are not functional for the most part.

Maybe the trace-log gives you some insight, [here is an example](https://github.com/sandboxie-plus/Sandboxie/issues/1208#issuecomment-1200170825). Or using [Procmon](https://learn.microsoft.com/en-us/sysinternals/downloads/procmon) (Process Monitor) to see what the process does, to see if it terminates or crashes.

It could be intentional, in which case the process detects being run sandboxed and terminates. Often this is done as DRM or as anti-cheat.
